app/views/guide_parts/edit.html.erb

```diff
@@ -0,0 +1,79 @@
+<% content_for :title_context, @edition.format %>
+<% content_for :page_title, @edition.title %>
+<% content_for :title, @edition.title %>
+
+<% if !@edition.errors.empty? || !@part.errors.empty? %>
+  <% content_for :error_summary do %>
+    <%= render("govuk_publishing_components/components/error_summary", {
+      id: "error-summary",
+      title: "There is a problem",
+      items: @part.errors.map do |error|
+        {
+          text: error.message,
+          href: "##{error.attribute.to_s}",
+        }
+      end,
+    }) %>
+  <% end %>
+<% end %>
+
+<%= render partial: "guide_parts/common_items", locals: { edition: @resource, reviewer: @reviewer } %>
+
+<div class="govuk-grid-row">
```

Comment:
We need to separate the heading and the following elements (`govuk-grid-column-two-thirds`, `options-sidebar`) into two separate `govuk-grid-row` containers. The reason is that otherwise the side panel is not sticky as it has no common parent. The second `govuk-grid-row` container should be inside the form.